client/laststand.lua

```diff
@@ -5,7 +5,8 @@ local sharedConfig = require 'config.shared'
 ---@param ped number
 function WaitForPlayerToStopMoving()
     local ped = cache.ped
-    while GetEntitySpeed(ped) > 0.5 or IsPedRagdoll(ped) do Wait(10) end
+    local TimeOut = 300
+    while GetEntitySpeed(ped) > 0.5 or IsPedRagdoll(ped) and TimeOut > 1 do TimeOut -= 1 Wait(10) end
```

Comment:
Maybe the timeout should be like 10 seconds instead of 3? Going through the windshield off of an overpass could take some time to fall down, bounce off of something and roll over.